common/src/test/java/gov/cms/ab2d/common/health/FileSystemCheckTest.java

```diff
@@ -0,0 +1,44 @@
+package gov.cms.ab2d.common.health;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FileSystemCheckTest {
+
+    @Test
+    void canWriteFile() throws IOException {
+        assertFalse(FileSystemCheck.canWriteFile(null, false));
+        assertTrue(FileSystemCheck.canWriteFile("/tmp", false));
+        assertTrue(FileSystemCheck.canWriteFile(".", false));
+        assertFalse(FileSystemCheck.canWriteFile("/notarealdir", false));
+        assertFalse(FileSystemCheck.canWriteFile("/bin", false));
```

Comment:
Is the idea this line will fail because the test isn't supposed to run as root?

Reply:
...I'd probably avoid trying to write to `/bin` (or any other system directory for that matter) in a test: depending on an individual developer's workstation configuration (permissions, etc), this write might actually succeed.

Reply:
How else do you get it to fail except to write to forbidden directories? I guess you could create a read only directory and then try. If the app can write to /bin, maybe you want to know that.

Reply:
Creating a R/O directory as you said might work. 

Another challenge with `/bin` is that it makes the test platform-dependent in some way, i.e Windows does not have bin.